Marlin/src/module/temperature.cpp

```diff
@@ -33,6 +33,13 @@
 #include "../core/language.h"
 #include "../HAL/shared/Delay.h"
 
+#if ENABLED(MAX6675_IS_MAX31865)
+  #include "Adafruit_MAX31865.h"
+  //Adafruit_MAX31865::Adafruit_MAX31865(int8_t spi_cs, int8_t spi_mosi, int8_t spi_miso, int8_t spi_clk)
+  //Adafruit_MAX31865 max31865 = Adafruit_MAX31865(65,63,42,40); // For software SPI set CS/MOSI/MISO/SCK
+  Adafruit_MAX31865 max31865 = Adafruit_MAX31865(49);// For hardware SPI, set CS pin
```

Comment:
49 ? Would be nice if this would be configurable and per channel.
max31865 ? Shouldn't this be an array for the channels?

Reply:
49 is an example, it worked in my case. guess it could be any.
For more channels on software SPI I think MOSI/MISO/SCK should be separately declared, then an array for CS.

Reply:
Changed them to overridable pin defines.